quasar-core/src/main/java/co/paralleluniverse/fibers/Fiber.java

```diff
@@ -2136,7 +2147,7 @@ public void write(Kryo kryo, Output output, Fiber<?> f) {
                 try {
                     f.stack.resumeStack();
                     kryo.writeClass(output, f.getClass());
-                    new FieldSerializer<>(kryo, f.getClass()).write(kryo, output, f);
+                    getFieldSerializer(kryo, f.getClass()).write(kryo, output, f);
```

Comment:
This is "original" Quasar code, and I don't know why Ron chose to create a new `FieldSerializer` each time :worried:.

If it's only the `SecurityManager` that makes this expensive then why should Corda 4 care?

Reply:
I think he assumed it was inexpensive.  

I care for C4 because it is clearly an unnecessary bit of reflection that happens and is an improvement and I happened to be set up for running the tests locally for this a bit better than I am for J11/J17 (and maybe alignment of code between versions is helpful).